save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -243,8 +245,8 @@ class FixPlugin(
         @Serializable(with = PathSerializer::class) val expected: Path
     ) : TestFiles {
         override fun withRelativePaths(root: Path) = copy(
-            test = test.relativeTo(root),
-            expected = expected.relativeTo(root),
+            test = test.createRelativePathToTheRoot(root).toPath(),
+            expected = expected.createRelativePathToTheRoot(root).toPath(),
```

Comment:
Can we cover this with any tests?

Reply:
I tried the `normalize`, but it changes nothing, about `canonicalize` - it will remove extra `..`, but it will also append relative path from the root like
```bash
Test1Test.java -> C:\dir1\dir2\Test1Test.java
```

`createRelativePathToTheRoot` actually covered by tests, but in future, when `.relativeTo` from okio will work suitable for us, may be we can not just change `createRelativePathToTheRoot` to the `relativeTo`, but put
`relativeTo` into `createRelativePathToTheRoot`, since the latest is the wrapper, which is covered by tests